app/build.gradle

```diff
@@ -88,6 +88,7 @@ android {
 
     kotlinOptions {
         jvmTarget = "1.8"
+        freeCompilerArgs += [ '-Xopt-in=kotlin.RequiresOptIn' ]
```

Comment:
Adding this option tells the Kotlin compiler that we want to support the `@OptIn` feature.  This basically allows code to use unstable APIs (e.g., coroutines) without requiring consumers of that code to also declare it.  i.e., it prevents propagation into the consumer code.